src/webviews/server.ts

```diff
@@ -81,3 +91,22 @@ export function registerWebviewServer(webview: vscode.Webview, commands: Protoco
 
     return { dispose: () => (messageListener.dispose(), disposeListeners()) }
 }
+
+/**
+ * Show an error to the user + button that links to the logged error
+ *
+ * @param err The error that was thrown in the backend
+ * @param webviewId Arbitrary value that identifies which webview had the error
+ * @param command The high level command/function that was run which triggered the error
+ */
+export function showLoggedErrorToUser(err: unknown, webviewId: string, command: string) {
+    // HACK: The following implementation is a hack, influenced by the implementation of handleError().
+    // The userFacingError message will be seen in the UI, and the detailedError message will provide the
+    // detailed information in the logs.
+    const detailedError = ToolkitError.chain(
+        err,
+        `The backend command "${command}()" failed when executed by the webview.`
+    )
+    const userFacingError = ToolkitError.chain(detailedError, 'A webview had an error.')
+    handleError(userFacingError, `webviewId="${webviewId}"`, 'A webview had an error.')
```

Comment:
Using passive voice makes it sound even more mysterious.

```suggestion
    handleError(userFacingError, `webviewId="${webviewId}"`, 'Webview error')
```